app/controllers/alerts_list_controller.rb

```diff
@@ -16,6 +17,15 @@ def index
     redirect_to :action => 'show'
   end
 
+  def class_icons
+    res = {}
+    icons.each do |k, _val|
+      file_name = "100/#{k.to_s.underscore}.png"
```

Comment:
this line would be better as `k.decorate.listicon_image` (assuming k is the class) .. but you'd need to add this method to the relevant decorators... so up to you.

But if you leave this like this, can you mention the list of classes that you're expecting to pass through here please? (So that it's easier to fix after.)